src/plinder/data/splits.py

```diff
@@ -53,43 +31,57 @@ class GraphConfig:
 
 @dataclass
 class SplitConfig:
-    proto_test_criteria: TestCriteria = field(default_factory=lambda: TestCriteria())
     graph_configs: list[GraphConfig] = field(
         default_factory=lambda: [
             GraphConfig("pli_qcov", 30, 1),
             GraphConfig("pocket_qcov", 50, 1),
         ]
     )
+    # how many unique congeneric IDs passing quality to consider as MMS
+    mms_unique_quality_count: int = 3
+    # what kind of cluster to use for sampling test
     test_cluster_cluster: str = "communities"
-    # metric to use for sampling representatives from each component
+    # metric to use for sampling representatives from each test cluster
     test_cluster_metric: str = "pli_qcov"
-    # threshold to use for sampling representatives from each component
+    # threshold to use for sampling representatives from each test cluster
     test_cluster_threshold: int = 50
-    # directed to use for sampling representatives from each component
+    # directed to use for sampling representatives from each test cluster
     test_cluster_directed: bool = False
+    # ?
     cluster_column: str = "cluster"
-    # max number of representatives from each community
+    # max number of representatives from each test cluster
     num_test_representatives: int = 3
     # test should not be singletons
     min_test_cluster_size: int = 5
-    # test/val should not be in too big communities or cause too many train cases to be removed
+    # test should not be in too big communities or cause too many train cases to be removed
     max_test_leakage_count: int = 300
-    mms_unique_quality_count: int = 3
+    # test should not have too few or too many interactions
+    min_max_test_pli: tuple[int, int] = (2, 50)
```

Comment:
@OleinikovasV we need to choose some values for min/max_test/val_pocket/pli. Suggest checking the cosmograph to see the distribution

Reply:
Cosmograph suggest there would be 371 418 systems within this range - which is a fairly permissive number; the key part is to avoid cases where only one interaction is counted.

I am a bit borderline on 2. Raising this to 3,50 - would drop 33 007 systems.

This might work still better with out 50% PLI splitting - avoiding the case of matching one of two PLIs, and counting that as leak.

Reply:
[a385a41](https://github.com/plinder-org/plinder/pull/14/commits/a385a41daf3c37a5adb09eb0b6a81d215cc0ed8d) - changed to (3, 50)